### ISSUE TITLE
Kraken Good Accessibility - Completed

### ISSUE THREAD

**Issue by OpenSZN39** [OpenSZN39/kraken-good-accessibility#2]
@thatkidmadrid

**Comment by thatkidmadrid** [comment]
- [ ] Some landmark roles are not on the correct elements; this ties into revisiting the structure of the page(you have both footers inside of a main)
- [x] The use of `<articles>` is not really correct, but we can ignore this for now as there are bigger fish to fry
- [x] `#` is not an acceptable description
- [x] I can tell you still haven't used VoiceOver on your website because something will sound very wrong if you tried using the navigation

**Comment by OpenSZN39** [comment]
navigation sounded fine when i used narrator on it.

**Comment by OpenSZN39** [comment]
Also if not articles what would be a proper tag to use?

**Comment by OpenSZN39** [comment]
Alright, I have made edits, based on your feedback from class. If you could explain in more detail some corrections I can focus on for the landmark roles, that would be fantastic.

**Comment by thatkidmadrid** [comment]
A few things are still a bit weird with the structure (the footer with the nav can be grouped with the main content and the header with the nav), but we can focus on fixing the landmark roles and call it a day.

The basic landmark roles are found in [the accessibility cheat sheet ](https://learn-the-web.algonquindesign.ca/topics/accessibility-cheat-sheet/#wai-aria-landmark-roles)
All content should be within aria-roles, and they should only appear once on each page. In cases where two components on the page could technically have the landmark role, you should decide which one is treated as the primary or more important version. For example, we have two navigations in this exercise: one in the header, one in the footer. We can decide that the one at the top is more important so although both are technically `<nav>` only the first one gets `role="navigation"` What I wanted to point out in your case is out of the two footer components, one of them more closely fits the description of the `contentinfo` role.

Here is how the [mozilla developer documentation](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Contentinfo_role) describes contentinfo:

> The contentinfo landmark role is used to identify information repeated at the end of every page of a website, including copyright information, navigation links, and privacy statements. This section is commonly called a footer.

So, instead of where you currently have the contentinfo role on the navigation footer, it should go with the footer with the copyright info

Make this change to your landmarks and I'll add the grade to LTW
